redirects-transition.conf

```diff
@@ -62,6 +62,32 @@ rewrite ^/ans/answers_public_funding.shtml https://www.fec.gov/introduction-camp
+rewrite ^/audits/2012/Oakland_County_Democratic_Party/VoteCertification-RequestforLegalConsiderationbytheCommission1317090.pdf https://www.fec.gov/resources/cms-content/documents/VoteCertification-RequestforLegalConsiderationbytheCommission1317090.pdf redirect;
+
+# audits/2010/ redirects
+rewrite ^/audits/2010/Democratic_Party_of_Illinois/CommitteeRequestforLegalConsiderationbyCommission1307079.pdf https://www.fec.gov/resources/cms-content/documents/CommitteeRequestforLegalConsiderationbyCommission1307079.pdf redirect;
+rewrite ^/audits/2010/Democratic_Party_of_Illinois/OfficeofGeneralCounselMemorandumtotheCommission-RequestforLegalConsiderationbytheCommission1307080.pdf https://www.fec.gov/resources/cms-content/documents/OGCMemorandumtotheCommission-RequestforLegalConsiderationbytheCommission1307080_-_Copy.pdf redirect;
+rewrite ^/audits/2010/Democratic_Party_of_Illinois/VoteCertification-RequestforLegalConsiderationbytheCommission1307081.pdf https://www.fec.gov/resources/cms-content/documents/VoteCertification-RequestforLegalConsiderationbytheCommission1307081.pdf redirect;
+rewrite ^/audits/2010/South_Dakota_Democratic_Party/CommitteeRequestforLegalConsiderationbytheCommission1315860.pdf https://www.fec.gov/resources/cms-content/documents/CommitteeRequestforLegalConsiderationbytheCommission1315860.pdf redirect;
+rewrite ^/audits/2010/South_Dakota_Democratic_Party/OfficeofGeneralCounselMemorandumtotheCommission-RequestforLegalConsiderationbytheCommission1315861.pdf https://www.fec.gov/resources/cms-content/documents/OGCMemorandumtotheCommission-RequestforLegalConsiderationbytheCommission1315861_-_Copy.pdf redirect;
+rewrite ^/audits/2010/South_Dakota_Democratic_Party/VoteCertification-RequestforLegalConsiderationbytheCommission1315862.pdf https://www.fec.gov/resources/cms-content/documents/VoteCertification-RequestforLegalConsiderationbytheCommission1315862.pdf redirect;
+
+# audits/2008/ redirects
+rewrite ^/audits/2008/RIGHTMARCH_COM_PAC_INC/CommitteeRequestforLegalConsiderationbyCommission1237843.pdf https://www.fec.gov/resources/cms-content/documents/CommitteeRequestforLegalConsiderationbyCommission1237843.pdf redirect;
+rewrite ^/audits/2008/RIGHTMARCH_COM_PAC_INC/SupplementalCommitteeRequestforLegalConsiderationbytheCommission1237844.pdf https://www.fec.gov/resources/cms-content/documents/SupplementalCommitteeRequestforLegalConsiderationbytheCommission1237844.pdf redirect;
+rewrite ^/audits/2008/RIGHTMARCH_COM_PAC_INC/OfficeofGeneralCounselMemo-CommitteeRequestforLegalConsiderationbytheCommission1237845.pdf https://www.fec.gov/resources/cms-content/documents/OfficeofGeneralCounselMemo-CommitteeRequestforLegalConsiderationbytheCommission1237845.pdf redirect;
+rewrite ^/audits/2008/RIGHTMARCH_COM_PAC_INC/VoteCertification-CommitteeRequestforLegalConsiderationbytheCommission1237846.pdf https://www.fec.gov/resources/cms-content/documents/VoteCertification-CommitteeRequestforLegalConsiderationbytheCommission1237846.pdf redirect;
+
```

Comment:
Per our offline discussion with @patphongs, please delete the audit redirects for now, since we're planning a big migration of all of them soon.